models/AddressesModel.php

```diff
@@ -26,7 +26,7 @@ class AddressesModel extends AbstractModel {
          * @return AddressEntity[]
          */
         public function getList(int $limit, int $offset):array {
-            $connection = DBConnectionProvider::getConnection(IDBConnection::class);
+            $connection = $this->connectToDB();
```

Comment:
Direct calls to DBConnectionProvider were replaced with calls of the new method.